### PR TITLE
Open nav after content type

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: BackofficeBundle/Resources/public/coffee/ContentTypeFormView.coffee [hunk 1]
```diff
@@ -1,8 +1,9 @@
 ContentTypeFormView = FullPageFormView.extend(
 
   onViewReady: ->
-    displayMenu() if @options.submitted
-
+    if @options.submitted
+      displayRoute = appRouter.generateUrl('listEntities', entityType: @options.entityType)
+      displayMenu(undefined, displayRoute)
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/public/coffee/ContentTypeFormView.coffee]: @itkg-nanne : Why don't you simply pass the displayRoute as the first argument ?

Comment (itkg-nanne) [inline BackofficeBundle/Resources/public/coffee/ContentTypeFormView.coffee]: to disable Backbone.history.navigate at the end of displayMenu() @itkg-ngilain

Comment (itkg-ngilain) [inline BackofficeBundle/Resources/public/coffee/ContentTypeFormView.coffee]: Ok, I see better now. I find weird to call a function with a route param sometimes as first argument, and sometimes as second, but in this case with no first parameter. Maybe you could change the function signature into something like ``displayMenu = (route, disableNavigate)``, with ``disableNavigate`` being a boolean.
That way the route is always the first parameter, and if required you can force to disable the ``Backbone.history.navigate``

**Comment by itkg-nanne** [comment]
ok for you @itkg-ngilain ?

**Comment by SOLenG** [comment]
:+1:

File: BackofficeBundle/Resources/public/coffee/orchestraLib.coffee [hunk 2]
```diff
@@ -29,12 +29,9 @@ opts =
...
 
-displayMenu = (route) ->
-  selectedPath = undefined
-  if typeof route isnt "undefined"
-    selectedPath = "#" + route
-  else
-    selectedPath = "#" + Backbone.history.fragment
+displayMenu = (route, refresh) ->
+  selectedPath = "#" + (route || Backbone.history.fragment)
+  refresh = refresh || (typeof route == "undefined")
```
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/public/coffee/orchestraLib.coffee]: you can use coffee typo : `` refresh = refresh ||  route?

Comment (itkg-nanne) [inline BackofficeBundle/Resources/public/coffee/orchestraLib.coffee]: will verify if route not null, won't work with undefined. tested @itkg-alavieille

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**